README_CN.md

```diff
@@ -46,26 +46,26 @@ Linkis 自2019年开源发布以来，已累计积累了700多家试验企业和
 
 # 下载
 
-请前往[Linkis releases 页面](https://github.com/WeBankFinTech/Linkis/wiki/Linkis-Releases) 下载Linkis 的已编译版本或源码包。
+请前往[Linkis releases 页面](https://github.com/apache/incubator-linkis/wiki/Linkis-Releases) 下载Linkis 的已编译版本或源码包。
```

Comment:
It is recommended to use [Linkis Releases Page](https://github.com/apache/incubator-linkis/releases)

Reply:
ok I will fix it